src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -22,11 +32,9 @@ const styles = StyleSheet.create({
   },
   textContainer: {
     backgroundColor: COLORS.lightgrey,
-    paddingLeft: 8,
-    paddingRight: 8,
     paddingTop: 4,
-    paddingBottom: 8,
-    borderRadius: 8,
+    paddingBottom: 4,
+    borderRadius: 12,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding and border radius values have been changed. Ensure these changes are consistent with the overall design language of your application.
```diff
-    paddingLeft: 8,
-    paddingRight: 8,
     paddingTop: 4,
-    paddingBottom: 8,
-    borderRadius: 8,
+    paddingBottom: 4,
+    borderRadius: 12,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->